roles/installer/templates/deployments/web.yaml.j2

```diff
@@ -387,6 +387,7 @@ spec:
           hostPath:
             path: /awx_devel
 {% endif %}
+{% endif %}
```

Comment:
Is there any reason to place `{% if development_mode | bool %}`? in `{% if projects_persistence|bool %}`?

Reply:
i think what @kurokobo is saying here is that in development mode project persistance can just be a hostPath mount and we should probably have a conditional for this

we can probably address this independently.
as always REALLY good catch!

@thedoubl3j can you open up a follow up issue to support project persistance in development mode